fdpa-resolutions.html

```diff
@@ -2,79 +2,21 @@
 <html>
   <head>
   <title>Fair Districts PA - Local Resolutions</title>
+  <meta name="referrer" content="origin">
   <style>
     #map-canvas { width:1100px; height:650px; }
-    .layer-wizard-search-label { font-family: sans-serif };
+    .layer-wizard-search-label { font-family: sans-serif }
+    .hidden { display:none; }
   </style>
-  <script type="text/javascript"
-    src="https://maps.google.com/maps/api/js?key=AIzaSyBxdPyKNXkcFbFPs5I_e-xqGCUPGVuyabs">
-  </script>
-  <script type="text/javascript">
-    var map;
-    var layer_0;
-    var layer_1;
-    function initialize() {
-      map = new google.maps.Map(document.getElementById('map-canvas'), {
-        center: new google.maps.LatLng(40.99058069667954, -77.62118299218751),
-        zoom: 8,
-        zoomControl: true,
-            zoomControlOptions: {
-            position: google.maps.ControlPosition.TOP_LEFT
-        },
-        fullscreenControl: true,
-        streetViewControl: false,
-        mapTypeControl: false
-      });
-      var style = [
-        {
-          featureType: 'all',
-          elementType: 'all',
-          stylers: [
-            { saturation: -85 }
-          ]
-        },
-        {
-          featureType: 'poi',
-          elementType: 'all',
-          stylers: [
-            { visibility: 'off' }
-          ]
-        }
-      ];
-      var styledMapType = new google.maps.StyledMapType(style, {
-        map: map,
-        name: 'Styled Map'
-      });
-      map.mapTypes.set('map-style', styledMapType);
-      map.setMapTypeId('map-style');
-      layer_0 = new google.maps.FusionTablesLayer({
-        query: {
-          select: "col12>>0",
-          from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
-          where: " col3>>0 = 'County'"
-        },
-        map: map,
-        styleId: 3,
-        templateId: 4
-      });
-      layer_1 = new google.maps.FusionTablesLayer({
-        query: {
-          select: "col16>>0",
-          from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
-          where: " col3>>0 in ('Borough', 'City', 'Town', 'Township (1st)', 'Township (2nd)', 'Organization') and col5>>0 in ('In work', 'Materials sent', 'Passed')"
-        },
-        map: map,
-        styleId: 4,
-        templateId: 5
-      });
-    }
-    google.maps.event.addDomListener(window, 'load', initialize);
-  </script>
   </head>
   <body>
     <h2>Fair Districts PA - Local Resolutions</h2>
-    <p>To date, more than 130 counties and municipalities across Pennsylvania have adopted a
+    <p id="fall-back-count">To date, more than 198 counties and municipalities across Pennsylvania have adopted a
         resolution in support of an indpendent citizens' redistricting commission.</p>

```

Comment:
Note: There's a merge conflict now because I fixed the spelling of "indpendent".  Should be trivial to resolve.

Reply:
Okay. With PRs, it is procedurally simpler to tell the author of the PR and let the author make the change, but given that neither of us is doing this f/t, maybe that stretches out the time period too much.

Reply:
I see, you made the fix in `master`. Merging it is no problem.